HomeWork17

MuravskyVadim opened the pull request.

MuravskyVadim committed:
HomeWork17

MuravskyVadim committed:
HomeWork17 added tag </form>

MuravskyVadim committed:
HomeWork17 added form for registration

Sofasmile commented on HomeWork01/src/servlets/Frontend.java:
```diff
@@ -0,0 +1,20 @@
+            resp.getWriter().println(value);
+        }
+
+        resp.setContentType("text/html;charset=utf-8");
```
try to make Strings which set some settings as constant

Sofasmile commented on HomeWork01/src/servlets/Frontend.java:
```diff
@@ -0,0 +1,20 @@
+import java.io.IOException;
+
+public class Frontend extends HttpServlet {
+    public void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
```
better use full name of the parameters

Sofasmile commented on HomeWork02/src/main/AccountServiceImpl.java:
```diff
@@ -0,0 +1,22 @@
+
+    @Override
+    public void singUp(String login, String password) {
+        if(login != null) {
```
we can use Objects.nonNull()

MuravskyVadim committed:
HomeWork17 next fixes:
1. Used few Strings as constants.
2. Used full name of the parameters.
3. Used other check on the null.

MuravskyVadim committed:
HomeWork17 next fixes:
1. Used few Strings as constants.
2. Used full name of the parameters.
3. Used other check on the null.
